plugins/kubernetes-backend/src/cluster-locator/GkeClusterLocator.test.ts

```diff
@@ -485,5 +486,20 @@ describe('GkeClusterLocator', () => {
         },
       ]);
     });
+    it('Check if new container.v1.ClusterManagerClient has key and values as parameter', async () => {
```

Comment:
Personally I'd want a test description that's a little more outcome-oriented like
```suggestion
    it('constructs ClusterManagerClient with identifying metadata', async () => {
```